serverless.yml

```diff
@@ -88,22 +88,40 @@ functions:
 
 # =--= MISC. AWS RESOURCES =-------------------------------------------------------------=
 
-# resources:
-#   Resources:
-#     usersTable:
-#       Type: AWS::DynamoDB::Table
-#       Properties:
-#         TableName: 52calls-52weeks-users
-#         KeySchema:
-#           - AttributeName: id
-#             KeyType: HASH
-#         AttributeDefinitions:
-#           - AttributeName: id
-#             AttributeType: S
-#         ProvisionedThroughput:
-#           ReadCapacityUnits: 5
-#           WriteCapacityUnits: 5
-#
+resources:
+  Resources:
+    usersTable:
+      Type: AWS::DynamoDB::Table
+      Properties:
+        TableName: 52calls-52weeks-users
```

Comment:
I *believe* we can change the name here to just `users` without loss of generality; lets give it a try.

I'm assuming serverless already does this, but we'll probably want to incorporate `${opt:stage}` into the name if it doesn't.  Once we get this to staging we can fast-follow to figure out if the name is scoped down somehow.